vhdbuilder/packer/vhd-image-builder-gen2.json

```diff
@@ -225,6 +225,16 @@
       "source": "parts/linux/cloud-init/artifacts/nvidia-device-plugin.service",
       "destination": "/home/packer/nvidia-device-plugin.service"
     },
+    {
```

Comment:
Hmm, is the intention for the the bind-mount unit files be present in the VHD? I see that you're also injecting it via nodecustomdata.

Reply:
yeah actually idk why I did both...

Reply:
probably fine in just custom data